tests/test_fxns.py

```diff
@@ -160,13 +159,15 @@ def test_find_png(vis_fxns):
     os.remove(test_file)
 
 
-def test_create_notebook(path_to_cif, vis_fxns):
+def test_create_notebook(ALA_CIF_file, vis_fxns):
+    path_to_cif = ALA_CIF_file
     result = vis_fxns.create_notebook(path_to_cif)
     path_to_notebook = path_to_cif.split(".")[0] + "_vis.ipynb"
     os.remove(path_to_notebook)
     assert result == "Visualization Complete"
 
 
+@pytest.mark.skip(reason="This has been deprecated")
```

Comment:
Why are we skipping these now?

Reply:
We dont use the SimulationFunctions class anymore since we move to multi-input tools :)

Reply:
then can we remove it>

Reply:
yes we can